src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -5,44 +5,44 @@
             [status-im.ui.components.icons.icons :as icons]
```

Comment:
if there are no changes except renaming better to revert this file, so we don't have changes in the old files